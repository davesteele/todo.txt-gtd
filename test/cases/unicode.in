# AProj

This item has a Ã‡edilla @nocontext

# _None
