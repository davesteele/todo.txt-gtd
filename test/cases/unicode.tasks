
@nocontext
This item has a Ã‡edilla @nocontext
