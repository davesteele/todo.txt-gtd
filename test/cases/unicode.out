# AProj

This item has a Ã‡edilla @nocontext +AProj

# _None
